tests/core/core_scope_test.py

```diff
@@ -235,5 +236,11 @@ def f(scope, x):
     with self.assertRaises(errors.LazyInitError):
       init_fn(random.PRNGKey(0), jax.ShapeDtypeStruct((8, 4), jnp.float32))
 
+  @temp_flip_flag('fix_rng', True)
+  def test_fold_in_static_seperator(self):
+    x = LazyRng(random.PRNGKey(0), ("ab", "c"))
+    y = LazyRng(random.PRNGKey(0), ("a", "bc"))
+    self.assertFalse(np.all(x.as_jax_rng() == y.as_jax_rng()))
```

Comment:
```suggestion
  # Create two LazyRng objects with different separators.
  x = LazyRng(random.PRNGKey(0), ("ab", "c"))
  y = LazyRng(random.PRNGKey(0), ("a", "bc"))

  # Check that the as_jax_rng() functions return different values.
  self.assertFalse(np.all(x.as_jax_rng() == y.as_jax_rng()))

  # Check that the fold_in_static_seperator() function returns a LazyRng object
  # with the correct separator.
  z = LazyRng.fold_in_static_seperator(x, "d")
  self.assertEqual(z.separator, ("ab", "cd"))
```

Reply:
Hi @Edwinvalerio, could you clarify what lines 250 and 251 do? `LazyRng` does not have a `fold_in_static_seperator` method or a `separator` attribute. The test will fail with those changes.